rust-src/concordium_base/src/id/types.rs

```diff
@@ -2470,6 +2470,19 @@ pub enum AccountCredentialValues<C: Curve, AttributeType: Attribute<C::Scalar>>
     },
 }
 
+pub trait HasAttributeCommitmentKey<C: Curve, TagType = AttributeTag> {
+    type ErrorType: 'static + Send + Sync + std::error::Error;
+
+    fn get_attribute_commitment_key(
+        &self,
+        attribute_tag: TagType,
+    ) -> Result<PedersenKey<C>, Self::ErrorType>;
+}
+
+pub trait HasBulletproofsGenerators<C: Curve> {
+    fn generators(&self) -> Generators<C>;
+}
```

Comment:
Do we actually use these traits anywhere? I can't find that, so please remove them unless I am missing something.

Reply:
I agree. I thought I hadn't pushed this.